sdk/ai/ai-projects/samples-dev/agents/file_search.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { AIProjectsClient, isOutputOfType, MessageContentOutput, MessageImageFileContentOutput, MessageTextContentOutput } from "@azure/ai-projects";
+import { delay } from "@azure/core-util";
+import { DefaultAzureCredential } from "@azure/identity";
+
+import * as dotenv from "dotenv";
+import * as fs from "fs";
+dotenv.config();
+
+const connectionString = process.env["AZURE_AI_PROJECTS_CONNECTION_STRING"] || "<endpoint>>;<subscription>;<resource group>;<project>";
+
+export async function main(): Promise<void> {
+  const client = AIProjectsClient.fromConnectionString(connectionString || "", new DefaultAzureCredential());
+
+  // Upload file
+  const localFileStream = fs.createReadStream("local_file.txt");
```

Comment:
Does the file get uploaded and immediately available for vector store ?

Reply:
yes it is immediately available